website/static/js/myProjects.js

```diff
@@ -263,14 +264,16 @@ function _formatDataforPO(item) {
     item.name = item.attributes.title;
     item.tags = item.attributes.tags.toString();
     item.contributors = '';
-    if (item.embeds.contributors.data){
+    var contributorsData = lodashGet(item, 'embeds.contributors.data', null);
+    if (contributorsData){
         item.embeds.contributors.data.forEach(function(c){
             var attr;
-            if (c.embeds.users.data) {
-                attr = c.embeds.users.data.attributes;
+            var users = lodashGet(c, 'embeds.users.data', null);
+            if (users) {
+                attr = users.attributes;
             }
             else {
-                attr = c.embeds.users.errors[0].meta;
+                attr = users.errors[0].meta;
```

Comment:
The error described in https://github.com/CenterForOpenScience/osf.io/pull/5454#issuecomment-208941401 seems to be happening here. We get to this branch when users are not embedded and there are no errors.

This should fix it:


```js
            var users = lodashGet(c, 'embeds.users.data', null);
            if (users) {
                if (users.errors) {
                    attr = users.errors[0].meta;
                } else {
                    attr = users.attributes;
                }
            }
            if (attr) {
                item.contributors += attr.full_name + ' ' + attr.middle_names + ' ' + attr.given_name + ' ' + attr.family_name + ' ' ;
            }
```